code/controllers/subsystem/shuttles/supply.dm

```diff
@@ -80,12 +80,14 @@
 	var/pointsEarned
 
 	for(var/atom/movable/MA in areaInstance)
-		SSshuttle.sold_atoms += " [MA.name]"
 		if(MA.anchored)	continue
+		SSshuttle.sold_atoms += " [MA.name]"
 
 		// Must be in a crate (or a critter crate)!
 		if(istype(MA,/obj/structure/closet/crate) || istype(MA,/obj/structure/closet/critter))
 			SSshuttle.sold_atoms += ":"
+			if(!MA.contents)
+				SSshuttle.sold_atoms += " (empty)"
```

Comment:
MA.contents never returns false even when the crate is empty, which means " (empty)" is never added.